recipes/lldb/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "5.0.0" %}
+{% set sha256 = "c0a0ca32105e9881d86b7ca886220147e686edc97fdb9f3657c6659dc6568b7d" %}
+
+package:
+  name: lldb
+  version: {{ version }}
+
+source:
+  fn: lldb-{{ version }}.src.tar.xz
+  url: http://releases.llvm.org/{{ version }}/lldb-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - build_external.patch
+
+build:
+  number: 0
+  skip: true  # [win and py<35]
+  features:
+    - vc14    # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.4.3
+    - ninja    # [win]
+    - python
+    - llvmdev =={{ version }}
+    - clangdev =={{ version }}
+    - libxml2 2.9.*
+    - swig
+    - libedit  # [linux]
+    - vc 14    # [win]
+    - libcxx   # [osx]
+  run:
+    - libedit  # [linux]
+    - vc 14    # [win]
+    - libcxx   # [osx]
+    - llvmdev =={{ version }}
+    - clangdev =={{ version }}
+    - python
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    - cmake >=3.4.3
+
+  commands:
+    - lldb --version
+
+about:
+  home: http://llvm.org/
+  license: NCSA
+  license_file: LICENSE.TXT
+  summary: The LLVM Debugger
+
+extra:
+  recipe-maintainers:
+    - inducer
```

Comment:
@inducer, are you okay with being a maintainer here?

Reply:
I guess.